opt_maven_install.sh

```diff
@@ -8,6 +8,7 @@ export MYSQL_VERSION=8.0.21
 if [[ "$1" == "true" ]]; then
 
 	yum -y install --setopt=skip_missing_names_on_install=False \
+		zip \
```

Comment:
I think we'll need to also install zip for the ART portion too, so we may want to update the yum install command here: https://github.com/kube-reporting/hive/blob/master/Dockerfile.rhel8#L3.

Reply:
roger